pyproject.toml

```
@@ -21,7 +21,7 @@ exclude = '''
 
 [tool.poetry]
 name = "scale-nucleus"
-version = "0.4.2"
+version = "0.4.3"
```

This is adding new features so semantic versioning says to bump minor 🙂 
```suggestion
version = "0.5.0"
```